packages/itwin/one-click-lca-widget/src/components/ExportModal.scss

```diff
@@ -0,0 +1,86 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@itwin/itwinui-css/scss/variables";
+
+.signin {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 4 * $iui-baseline;
+  padding-top: 2 * $iui-baseline;
+
+  &-icon {
+    width: 2 * $iui-icons-3xl;
+    height: 1 * $iui-icons-3xl;
+  }
+
+  &-prompt {
+    margin: $iui-l 0;
+    font-size: medium;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  &-form {
+    width: 70%;
+  }
+
+  &-input {
+    width: 100%;
+    margin: $iui-xs 0;
+  }
+
+  &-button-container {
+    margin: $iui-sm 0;
+  }
+
+  &-button {
+    width: 100%;
+  }
+
+  &-error {
+    height: $iui-xl;
+    font-size: small;
+  }
+
+  &-wait {
+    vertical-align: middle;
+    line-height: normal;
+  }
+}
+
+.iui-text-leading {
```

Comment:
i worry this might break consumers if this is an actual class coming from itwinui

Reply:
please update some of the generic class names here so they dont collide